CONTRIBUTING.md

```diff
@@ -229,13 +229,16 @@ Each source file needs to have this header:
     // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
     // See the License for the specific language governing permissions and
     // limitations under the License.
+    //
+    // SPDX-License-Identifier: Apache-2.0
 ```
-Note: `DATE` is either a year or a range of years with the first and last years of the range separated by a comma. So for example: "2004" or "2000, 2004". The first year is when the contents of the file were first created and the last year is when the contents were last modified.
+Note: The date is either a year or a range of years with the first and last years of the range separated by a dash. For example: "2004" (initial and last contribution in the same year) or "2000 - 2004". The first year is when the contents of the file were first created and the last year is when the contents were last modified. The years of contribution should be ordered in chronological order, thus the last date in the list should be the year of the most recent contribution. If there is a gap between contributions of one or more calendar years, use a comma to separate the disconnected contribution periods (e.g. "2000 - 2004, 2006").
 
 Example:
 
-```
-    // Copyright (c) 2018, 2020 by ACME Corp, Globex. All rights reserved.
+```cpp
+    // Copyright (c) 2019 - 2020, 2022 by Acme Corporation. All rights reserved.
+    // Copyright (c) 2020 - 2022 by John Doe. All rights reserved.
```

Comment:
an E-Mail address might be nice in order to be able to contact the copyright holder if it's a private person and the file was copied to a 3rd party repo
```
// Copyright (c) 2020 - 2022 by John Doe <john@doe.xyz>. All rights reserved.
```
Like in this example with Jane Doe https://spdx.github.io/spdx-spec/appendix-IX-file-tags/